2015/9-easy-exceptions-and-logging.pod

```diff
@@ -0,0 +1,293 @@
+=encoding utf8
+
+=head1 Exception handling, error logging and user messages made easy
+
+=head2 Introduction
+
+Have you ever experienced any of the following when writing your Dancer
+application?
+
+=over 1
+
+=item * Found that handling user errors involved writing a lot of code
+
+=item * Wanted to log messages from a module, without that module depending on Dancer2
+
+=item * Wanted flexibility on sending messages to the user, to the system, or both
+
+=item * Found yourself writing the same code in every application in order to send messages to a template
+
+=item * Wanted a clean way to handle 500 errors, whilst getting notified as the system administrator
+
+=back
+
+L<Dancer2::Plugin::LogReport> combined with L<Dancer2::Logger::LogReport>
+are designed to do all of the above, and more.
+Although the 2 modules can be used separately, it's recommended to use
```

Comment:
I think it's actually meant to be one paragraph, so I've removed the line break altogether.